docker-compose.yml

```diff
@@ -60,6 +61,24 @@ services:
     networks:
       - my-network
 
+  centops:
+    build:
+      context: ../CentOps/src
+      dockerfile: ./CentOps.Api/Dockerfile
+    ports:
+      - "9014:80"
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=http://+:80
+      - COSMOSDB__ACCOUNT=
```

Comment:
You can probably remove any reference to Cosmos in here if you're InMemory to reduce confusion.

Reply:
Done